airbyte-integrations/connectors/source-instagram/source_instagram/streams.py

```diff
@@ -251,7 +251,7 @@ def _state_has_legacy_format(self, state: Mapping[str, Any]) -> bool:
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]):
         """Update stream state from latest record"""
-        record_value = latest_record[self.cursor_field]
+        record_value = latest_record.get(self.cursor_field, self._start_date)
```

Comment:
As Denys suggested: please bump this connector version as you changed its code.

Reply:
Fix to instagram deleted. Resolved.